pkg/userdata/flatcar/provider.go

```diff
@@ -374,6 +376,13 @@ storage:
         inline: |
 {{ .NodeIPScript | indent 10 }}
 
+    - path: "/etc/systemd/network/zz-default.network.d/ipv6-fix.conf"
+      filesystem: root
+      mode: 0755
+      contents:
+        inline: |
+{{ .IPv6RouterRA | indent 10 }}
```

Comment:
Do we want to apply this unconditionally? Can we use [static-nic.network]( https://github.com/kubermatic/machine-controller/blob/master/pkg/userdata/flatcar/provider.go#L176) for this?

Reply:
> Do we want to apply this unconditionally?

Yes. Because it harms nothing. We could apply it only in IPv6 or IPv4+IPv6 mode but not worth adding complexity to the code I think.

> Can we use [static-nic.network](https://github.com/kubermatic/machine-controller/blob/master/pkg/userdata/flatcar/provider.go#L176) for this?

Afaik it should work even if we put it there.

It will apply only to the interfaces with names matching `en*`.   So I put it here where it applies to any interface.

Reply:
Okay makes sense 👍